apps/torus-bridge/src/app/_components/transfer-evm.tsx

```
@@ -47,55 +74,86 @@ export function TransferEVM() {
   const { transfer, selectedAccount, isInitialized, isAccountConnected, api } =
     useTorus();
   const { data: walletClient } = useWalletClient();
   const { chain, address } = useAccount();
   const { switchChain } = useSwitchChain();
-  const { toast } = useToast();
 
-  const evmSS58Addr = userInputEthAddr
-    ? convertH160ToSS58(userInputEthAddr)
-    : "";
-  const amountRems = amount ? toNano(parseFloat(amount)) : BigInt(0);
+  const getChainValues = getChainValuesOnEnv(
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getChainValues'. Consider to delete it or rename it.

'getChainValues' is defined here:


(lint/suspicious/noRedeclare)